lizard_efcis/import_data.py

```diff
@@ -709,11 +719,12 @@ def manual_import_csv(self, filename, mapping_code,
                         continue
                     else:
                         logger.error(ex.message)
-                        action_log.update({"IntegrityError%s" % count: ex.message})
+                        action_log.update(
+                            {"IntegrityError%s" % count: ex.message})
```

Comment:
Ja, dit is een geldige, 
"IntegrityError%s" % count: - vormen sammen een key
ex,message - een waarde